xmantissa/test/test_stats.py

```diff
@@ -59,10 +59,9 @@ def test_deliverStatEvents(self):
                 (d['key'], d['value'])
                 for d in parseString(stat['data'])
                 if d['key'] != 'time'])
-        self.assertEqual(
-            data,
-            set([('foo', 'bar'), ('baz', '12'),
-                 ('quux', u'\N{SNOWMAN}'.encode('utf-8'))]))
+        for item in set([('foo', 'bar'), ('baz', '12'),
```

Comment:
Tests shouldn't have control flow in them.  Rather than a for loop, could you make this a series of `assertIn` statements, so that when one fails, it's clearer from looking at the traceback *which* one failed?